src/app/container/view/view.component.ts

```diff
@@ -83,7 +83,18 @@ export class ViewContainerComponent extends View implements OnInit {
   }
 
   deleteHostedTag(): void {
-    const deleteMessage = 'Are you sure you want to delete tag ' + this.version.name + ' for tool ' + this.tool.tool_path + '?';
+    let deleteMessage;
+    if (this.defaultVersion === this.version.name) {
+      deleteMessage =
+        'You are trying to delete the default version ' +
+        this.defaultVersion +
+        ' for the tool ' +
+        this.tool.tool_path +
+        '. Deleting this will set the default version to be the latest version. Are you sure you want to continue?';
+    } else {
+      deleteMessage = 'Are you sure you want to delete tag ' + this.version.name + ' for tool ' + this.tool.tool_path + '?';
```

Comment:
This is fine,  was like this, and is a matter of preference, but FYI, in TypeScript you can use string interpolation, so you don't have to have the `+` signs (surround with backticks instead of single/double quotes)

```
deleteMessage = `Are you sure you want to delete tag ${this.version.name} for tool ${this.tool.tool_path}?'
```